Symfony/CS/Fixer/Contrib/EchoToPrintFixer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EchoToPrintFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $echoTokens = $tokens->findGivenKind(T_ECHO);
+
+        foreach ($echoTokens as $echoIndex => $echoToken) {
+            $nextTokenIndex = $tokens->getNextMeaningfulToken($echoIndex);
+            $endTokenIndex = $tokens->getNextTokenOfKind($echoIndex, array(';'));
```

Comment:
don't read whole logic, but sth like:
`<div><?php echo XXX ?></div><div><?php echo YYY; ?></div>`
under `XXX` put echo statement that may be changed, and unde `YYY` put echo statement that can't be changed. Please notice that after `XXX` there are no `;`, and it is after `YYY`.
The current `getnextTokenOfKind` call will match the `;` after `YYY`, which will have an influence if the first `echo` could be converted or no